factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/PgConfigurationProperties.java

```diff
@@ -80,6 +80,12 @@
      */
     int queueFetchRatio = 4;
 
+    URL schemaRegistryUrl;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4877539)